File: CompilationTest/FakeEnumTest/results/FakeEnumTest.swift
class TokenType {
  static var endToken: String = "EndToken"
  static var whitespace: String = "Whitespace"
  static var identifier: String = "Identifier"
  static var operatorX: String = "Operator"
  static var noInitializer: String
}

let casingTest = TokenType.endToken
print(casingTest)

File: CompilationTest/FakeEnumTest/results/FakeEnumTest.rb
class TokenType
  @end_token = "EndToken"
  @whitespace = "Whitespace"
  @identifier = "Identifier"
  @operator_x = "Operator"
  @no_initializer = nil

  class << self
    attr_accessor :end_token, :whitespace, :identifier, :operator_x, :no_initializer
  end
end

casing_test = TokenType.end_token
puts casing_test